src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -75,6 +75,7 @@ def load_arguments(self, _):
         c.argument('kube_context', options_list=['--kube-context'], arg_group='Target Kubernetes Cluster', help='Kubconfig context from current machine.')
         c.argument('storage_account', options_list=['--storage-account'], arg_group='Storage Account', help='Name or ID of the storage account to save the diagnostic information')
         c.argument('sas_token', options_list=['--sas-token'], arg_group='Storage Account', help='The SAS token with writable permission for the storage account.')
+        c.argument('container_name', options_list=['--container-name'], help = 'The name of the storage container.')
```

Comment:
make it a part of **_arg_group='Storage Account'_**

As in line 77.

Reply:
Lets not make container name a required parameter.

Probably keep some default name as well, like "connectedk8s_troubleshoot_logs" or something